### PR TITLE
fixed use default column issue for dlgName

### PR DESCRIPTION
**Pull request by dannyparsons**

### CODE REVIEW

File: instat/dlgName.vb [hunk 1]
```diff
@@ -17,6 +17,7 @@
 Imports instat.Translations
 Public Class dlgName
     Dim bFirstLoad As Boolean = True
+    Dim bUseSelectedColumn As Boolean = False
```
Comment (dannyparsons) [inline instat/dlgName.vb]: @deaspo boolean to say if we should set a default column on Load.

File: instat/dlgName.vb [hunk 2]
```diff
@@ -27,6 +28,10 @@ Public Class dlgName
         ucrBase.clsRsyntax.SetFunction(frmMain.clsRLink.strInstatDataObject & "$rename_column_in_data")
         If bFirstLoad Then
             SetDefaults()
+            bFirstLoad = False
+        End If
+        If bUseSelectedColumn Then
+            SetDefaultColumn()
```
Comment (dannyparsons) [inline instat/dlgName.vb]: Call this after SetDefaults() if boolean is True.

File: instat/dlgName.vb [hunk 3]
```diff
@@ -35,18 +40,18 @@ Public Class dlgName
...
     Public Sub SetCurrentColumn(strColumn As String, strDataFrame As String)
         strSelectedColumn = strColumn
         strSelectedDataFrame = strDataFrame
+        bUseSelectedColumn = True
+    End Sub
+
+    Private Sub SetDefaultColumn()
+        ucrSelectVariables.ucrAvailableDataFrames.cboAvailableDataFrames.SelectedItem = strSelectedDataFrame
+        ucrReceiverName.SetSelected(strSelectedColumn, strSelectedDataFrame)
+        bUseSelectedColumn = False
```
Comment (dannyparsons) [inline instat/dlgName.vb]: @deaspo Set the values of the data frame selector and receiver.

Comment (deaspo) [inline instat/dlgName.vb]: Okay, understood

**Pull request merged**